src/pyggdrasil/distances/_interface.py

```diff
@@ -7,8 +7,8 @@
 _IntegerTreeRoot: TypeAlias = TreeNode[int, Any]
 
 
-class TreeSimilarity(Protocol):
-    """Interface for similarity functions between the trees.
+class TreeSimilarityMeasure(Protocol):
```

Comment:
I wanted to have once common type or both `TreeDistance` and `TreeSimilarity` -  let me know what you think of the design.